src/js/structs/Item.js

```diff
@@ -14,4 +14,8 @@ module.exports = class Item {
 
     return this._itemData[key];
   }
+
+  toJSON() {
+    return this._itemData;
```

Comment:
Well, we are on the `Item` class, and we are aware of the private properties of the class. In this particular case, I think `this.get()` will only add an extra useless call?